src/components/WorkDescription/ProblemSolution.js

```diff
@@ -12,15 +12,34 @@ const Content = (props) => (
     <div className="content" {...props} />
     <style jsx global>{`
       .content * + * {
-        margin-top: 1.5em;
+        margin: 0;
+        padding: 0;
+        border: none;
       }
 
       .content li + li {
```

Comment:
If you don't need the margin between the list elements anymore you could remove this bit of css ;)